release/tools/vmimage.subr

```diff
@@ -88,6 +88,9 @@ vm_install_base() {
 }
 
 vm_emulation_setup() {
+	if [ -n "${WITHOUT_QEMU}" ]; then
+		return 0
+	fi
 	if ! [ -z "${QEMUSTATIC}" ]; then
 		export EMULATOR=/qemu
 		cp ${QEMUSTATIC} ${DESTDIR}/${EMULATOR}
```

Comment:
It might be worth adding a new newaliases rc.d script conditional on firstboot to run newaliases if needed?

Hmm, looks like /etc/rc.d/sendmail already runs newaliases if needed, but only if sendmail is enabled.  I wonder if that part should just move out to a dedicated /etc/rc.d/newaliases script.

Reply:
I've written one (https://reviews.freebsd.org/D49309) but it turns out to be mostly pointless because the in the default install newaliases uses dma which doesn't build an aliases.db at all (it just validates its arguments and returns 0).